scripts/setup_mysql

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Usage: setup_mysql <database_name>
+# expected ENV: MYSQL_HOST, MYSQL_USER, MYSQL_PASSWORD
+# Description:
+# Setup the database if it doesn't exist.
+
+if echo "use $1; show tables" | mysql --host=${MYSQL_HOST:-mysql} --user=${MYSQL_USER:-root} --password=${MYSQL_PASSWORD:-root} | grep schema_migrations 2>&1 > /dev/null
+then
+  echo "🐋 Database exists, doing nothing."
```

Comment:
A 🐬 would fit much better here. This is serious stuff! ;-)
https://www.google.ch/search?q=mysql&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjw_JjOypHSAhXIqxoKHWDnA90Q_AUICCgB&biw=1570&bih=910&dpr=2